graphbolt/CMakeLists.txt

```diff
@@ -23,10 +23,6 @@ list(GET TORCH_PREFIX_VER 1 TORCH_VER)
 message(STATUS "Configuring for PyTorch ${TORCH_VER}")
 string(REPLACE "." ";" TORCH_VERSION_LIST ${TORCH_VER})
 
-if(USE_CUDA)
```

Comment:
Split out to a different PR, unrelated to your code.

Reply:
sure, remove for fix